src/java/org/apache/cassandra/db/marshal/UserType.java

```diff
@@ -384,15 +412,19 @@ private boolean equalsWithoutTypes(UserType other)
         return name.equals(other.name)
             && fieldNames.equals(other.fieldNames)
             && keyspace.equals(other.keyspace)
-            && isMultiCell == other.isMultiCell;
+            && isMultiCell == other.isMultiCell
+            && comment.equals(other.comment)
+            && securityLabel.equals(other.securityLabel);
```

Comment:
Similar reasoning with `ColumnSpec`, I do not think comment and securityLabel should be part of equals.